lib/vintage_net_qmi/connectivity.ex

```diff
@@ -93,6 +95,32 @@ defmodule VintageNetQMI.Connectivity do
     {:noreply, new_state}
   end
 
+  defp update_time_location_properties(serving_system, state) do
+    fields = [
+      :cell_id,
+      :location_area_code,
+      :network_datetime,
+      :roaming,
+      :timezone_offset,
+      :daylight_saving_adjustment
+    ]
```

Comment:
Just a nit, but maybe this is better off as a module attribute to save some assignments?

Reply:
Yeah, I thought about that.  I went in this direction because I am not using this list anywhere else and it is nice to have this information local to where it is being used. I could do module attribute right above this function or I could pipe the list directly into the `Enum.each/2` call if the assignment is something you are looking to get rid of?

I am fine either way as they all lead to the same output, so if you have a strong preference towards the module attribute I can update.

Reply:
Na, dont worry about it